src/client/test_utils.rs

```diff
@@ -330,7 +331,12 @@ impl MicrogridApiClient for MockMicrogridApiClient {
                     for metrics in metrics.iter() {
                         interval.tick().await;
                         next_ts += dur;
-                        let ts = Some(prost_types::Timestamp::from(next_ts));
+                        let duration_since_epoch =
+                            next_ts.duration_since(SystemTime::UNIX_EPOCH).unwrap();
+                        let ts = Some(protobuf::Timestamp {
+                            seconds: duration_since_epoch.as_secs() as i64,
+                            nanos: duration_since_epoch.subsec_nanos() as i32,
+                        });
```

Comment:
Is this fixing a bug?  If so, might be worth putting into its own commit, so you can document it.  If it is just adjusting to some API change without any changes to behaviour, it can stay here.